### ISSUE TITLE
Shape of labels does not match shape of predictions

### ISSUE THREAD

**Issue by PkuRainBow** [deepinsight/insightface#116]
I try to try the loss functions on other datasets and test the basic softmax loss functions, but it tells me that 👍 

> Shape of labels does not match shape of predictions

I will update more details latter.
